src/attributes/attr.js

```diff
@@ -3,11 +3,14 @@ define([
 	"../core/access",
 	"./support",
 	"../var/rnotwhite",
+	"./val",
```

Comment:
I wonder, we don't have it in `master` plus it seems we don't use this declaration, test doesn't fail without it. Maybe it is needed to build this module and that is indirect dependency?

@timmywil? We can deal with this later though